src/solver/optimisation/adequacy_patch_csr/hourly_csr_problem.h

```diff
@@ -72,9 +77,14 @@ class HourlyCSRProblem
     double belowThisThresholdSetToZero;
     PROBLEME_HEBDO* problemeHebdo_;
     PROBLEME_ANTARES_A_RESOUDRE problemeAResoudre_;
-    explicit HourlyCSRProblem(PROBLEME_HEBDO* p) : problemeHebdo_(p)
+
+    explicit HourlyCSRProblem(AdqPatchParams& adqPatchParams,PROBLEME_HEBDO* p) :
+        adqPatchParams_(adqPatchParams),
```

Comment:
Can we go one step further and pass `adqPatchParams.curtailmentSharing` here ?

Reply:
Not really because of : 
`adq_patch_curtailment_sharing.cpp(134):adqPatchParams_.localMatching.setToZeroOutsideInsideLinks,`